### PR TITLE
Added routes to url.

### PR DESCRIPTION
**Pull request by send-on**
Please review.

Change:
- Added routes to url to go to different restaurants.

### CODE REVIEW

File: client/components/Description.jsx [hunk 1]
```diff
@@ -17,14 +17,15 @@ export default class Description extends React.Component {
   }
   componentDidMount() {
     this.fetchReviews();
-
   }
 
   fetchReviews() {
     let context = this;
-    axios.get('/api')
+    let id = window.location.href.split('/')[4]
```
Comment (tassteven) [inline client/components/Description.jsx:24]: Smart way of extracting the id from the url!

File: server/server.js [hunk 2]
```diff
@@ -13,25 +13,26 @@ app.use(bodyParser.urlencoded({ extended: false }))
 
 app.use(bodyParser.json())
 
-app.get('/restaurants/:id', function(req, res) {
-  var place_id = 'req.params.id'
 
+app.use('/restaurants', express.static(path.join(__dirname, '../public')));
+
+app.get('/restaurants/:id', function(req, res) {
+  res.sendFile(path.join(__dirname, '../public/index.html'))
```
Comment (tassteven) [inline server/server.js:20]: Interesting way to render page according id by two get requests! nice!

**Review by tassteven — APPROVED** [review]
Overall excellent job and clever way to dynamically render your components according to the id in the URL.
Suggestion:
- Make sure to add error handling when user inputs incorrect or no id in the url.

**Comment by send-on** [comment]
Thanks for the insightful review Mr. Steven Chiang!

**Pull request merged**